measures/BuildExistingModel/measure.rb

```diff
@@ -225,7 +225,7 @@ def run(model, runner, user_arguments)
       measures['BuildResidentialHPXML'][0][step_value.name] = value
     end
 
-    measures['HPXMLtoOpenStudio'] = [{ 'hpxml_path' => File.expand_path('../existing.xml') }]
+    measures['HPXMLtoOpenStudio'] = [{ 'hpxml_path' => File.expand_path('../existing.xml'), 'output_dir' => File.expand_path('..'), 'debug' => true }]
```

Comment:
We should change the HPXMLtoOpenStudio measure so that you don't have to set debug=true just to get the in.xml file, since that should have some runtime impact. Either we should A) always generate in.xml (this is my preference assuming the added runtime is really small) or B) create a new measure argument specifically asking if this file should be generated.